src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts

```diff
@@ -20,13 +21,35 @@ import "../../components/hui-entity-editor";
 import "../../../../components/ha-card";
 import "../../../../components/ha-icon";
 
+const entitiesConfigStruct = struct.union([
+  {
+    entity: "entity-id",
+    name: "string?",
+    icon: "icon?",
+  },
+  "entity-id",
+]);
+
+const cardConfigStruct = struct({
+  type: "string",
+  id: "string|number",
+  title: "string|number?",
```

Comment:
Why is number an option here?

Reply:
Title is optional, and can be either string or number, because technically a number is also a good title.

Reply:
YAML is to blame here. If you want the title to be 23, it will detect it's a number and store it as a number. So accepting number is good.